lib/Dancer2/Plugin/Auth/Extensible.pm

```diff
@@ -462,6 +462,9 @@ sub authenticate_user {
     my ($dsl, $username, $password, $realm) = @_;
     my @realms_to_check = $realm? ($realm) : (keys %{ $settings->{realms} });
 
+    warn "No Auth::Extensible realms configured with which to authenticate user"
+        unless @realms_to_check;
+
```

Comment:
Hmmm, this could actually simply be ````or warn "No Auth::Extensible realms...````

I won't change it though unless you want me to, to prevent having multiple commits or needing to force push.